lib/events.dart

```diff
@@ -27,17 +30,14 @@ abstract class IProovEvent {
       case 'processing':
         return IProovEvent.progress(map['progress'], map['message']);
       case 'success':
-        Image? frame;
-        if (map['frame'] != null) {
-          frame = decodePng(map['frame']!);
-        }
+        final frameData = map['frame'];
+        final frame = frameData != null ? decodePng(frameData) as Image : null;
```

Comment:
Slightly cleaner. `as Image` is an explicit cast (usually recommended by linter)